### PR TITLE
MySQL workspace service

### PR DESCRIPTION
**Pull request by anatbal**
# PR for issue #1844 

## What is being addressed
 
A mysql workspace service

### CODE REVIEW

File: templates/workspace_services/mysql/Dockerfile.tmpl [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM debian:buster
+
+# Install Azure CLI
+RUN apt-get update \
```
Comment (marrobi) [inline templates/workspace_services/mysql/Dockerfile.tmpl:4]: Is this needed?

File: templates/workspace_services/mysql/terraform/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+# This script assumes you have created an .env from the sample and the variables
+# will come from there.
+# shellcheck disable=SC2154
+export TF_VAR_docker_registry_server="$TF_VAR_acr_name.azurecr.io"
+export TF_VAR_docker_registry_username=$TF_VAR_acr_name
+TF_VAR_docker_registry_password=$(az acr credential show --name "${TF_VAR_acr_name}" --query passwords[0].value -o tsv | sed 's/"//g')
```
Comment (marrobi) [inline templates/workspace_services/mysql/terraform/deploy.sh:9]: Is this needed?

File: templates/workspace_services/mysql/terraform/destroy.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+
+# This script assumes you have created an .env from the sample and the variables
+# will come from there.
+# shellcheck disable=SC2154
```
Comment (marrobi) [inline templates/workspace_services/mysql/terraform/destroy.sh:6]: Is this needed.

**Review by marrobi — APPROVED** [review]
LGTM, couple of comments.

**Review by ross-p-smith — APPROVED** [review]

File: templates/workspace_services/mysql/terraform/main.tf [hunk 4]
```diff
@@ -0,0 +1,46 @@
+# Azure Provider source and version being used
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.97.0"
```
[the remark this answers is not in the document]
Comment (ross-p-smith) [reply templates/workspace_services/mysql/terraform/main.tf:6]: There's another PR moving this to 3.5.0 - So I would settle on that.

**Pull request closed without merging**